client/lib/cart-values/index.js

```diff
@@ -133,5 +137,6 @@ module.exports = {
 	getRefundPolicy,
 	isFree,
 	isPaidForFullyInCredits,
-	isPayPalExpressEnabled
+	isPayPalExpressEnabled,
+	isCreditCardPaymentsEnabled: isCreditCardPaymentsEnabled
```

Comment:
I am blind :D All the other rows around use that syntax and I don't see that the one added is different..